.gitignore

```diff
@@ -1,5 +1,6 @@
 /Local.mk
 
+*.tgz
```

Comment:
Is this specifically for build/configcli.tgz? Maybe we should be more specific.

Reply:
Yes it is, do we want to put any tgz ?

Reply:
Dunno, I'm a little skittish about really broad ignore rules because they can result in inadvertent failures to check in files in the future.

Reply:
ok sure